scm-core/src/main/java/sonia/scm/repository/work/SimpleWorkingCopyFactory.java

```diff
@@ -205,19 +217,23 @@ public Repository getScmRepository() {
 
     private void close(ParentAndClone<R, W> parentAndClone) {
       try {
-        closeWorkingCopy(parentAndClone.getClone());
-      } catch (Exception e) {
-        LOG.warn("could not close clone for {} in directory {}", getScmRepository(), parentAndClone.getDirectory(), e);
-      }
-      try {
-        closeRepository(parentAndClone.getParent());
-      } catch (Exception e) {
-        LOG.warn("could not close central repository for {}", getScmRepository(), e);
-      }
-      try {
-        workingCopyPool.contextClosed(this, parentAndClone.getDirectory());
-      } catch (Exception e) {
-        LOG.warn("could not close context for {} with directory {}", getScmRepository(), parentAndClone.getDirectory(), e);
+        try {
+          closeWorkingCopy(parentAndClone.getClone());
+        } catch (Exception e) {
+          LOG.warn("could not close clone for {} in directory {}", getScmRepository(), parentAndClone.getDirectory(), e);
+        }
+        try {
+          closeRepository(parentAndClone.getParent());
+        } catch (Exception e) {
+          LOG.warn("could not close central repository for {}", getScmRepository(), e);
+        }
+        try {
+          workingCopyPool.contextClosed(this, parentAndClone.getDirectory());
+        } catch (Exception e) {
+          LOG.warn("could not close context for {} with directory {}", getScmRepository(), parentAndClone.getDirectory(), e);
+        }
```

Comment:
I think it would be much more readable if we move the work out in to its own method e.g.: `closeResources`.